src/commands/PublishCommand.js

```diff
@@ -213,6 +213,10 @@ export default class PublishCommand extends Command {
     const patch = semver.inc(currentVersion, "patch");
     const minor = semver.inc(currentVersion, "minor");
     const major = semver.inc(currentVersion, "major");
+    const prepatch = semver.inc(currentVersion, "prepatch");
+    const preminor = semver.inc(currentVersion, "preminor");
+    const premajor = semver.inc(currentVersion, "premajor");
+    const prerelease = semver.inc(currentVersion, "prerelease");
```

Comment:
Should it prompt you for an identifier if one already exists? Or maybe it prompts you but hitting enter will use the existing identifier.